submissions/Roophee/a-tiny-js-world-oop/index.js

```diff
@@ -0,0 +1,68 @@
+"use strict";
+
+const delimiter = '; '
+
+class Inhabitant {
+    constructor(name, gender, saying, legs = 4, tail = 1) {
+        this.name = name;
+        this.gender = gender;
+        this.species = this.constructor.name.toLowerCase();
+        this.saying = saying;
+        this.legs = legs;
+        this.tail = tail;
+        this.friends = [];
+    }
+
+    addFriends(...arrayOfFriends) {
+        arrayOfFriends.forEach(({ name }) => this.friends.push(name));
+    }
+
+    description() {
+        return ['name', 'gender', 'species', 'saying', 'legs', 'tail', 'friends'].map(property => Array.isarray(this[property]) && this[property].length === 0 ? `${property}:none` : `${property}:${this[property]}`).join(delimiter);
```

Comment:
```suggestion
        return ['name', 'gender', 'species', 'saying', 'legs', 'tail', 'friends'].map(property => `${property}: ` + Array.isarray(this[property]) && this[property].length === 0 ? 'none' : this[property]).join(delimiter);
```
or
```suggestion
        return ['name', 'gender', 'species', 'saying', 'legs', 'tail', 'friends'].map(property => `${property}: ${Array.isarray(this[property]) && this[property].length === 0 ? 'none' : this[property]}`).join(delimiter);
```

to avoid duplication of `` `${property}: ` ``

Reply:
Thanks, it is height level art of codding).
Thanks for this experience and knowledge!!!